lib/plugin-compat.php

```diff
@@ -25,10 +25,12 @@
  * @param  array $post      Post object which contains content to check for block.
  * @return array $post      Post object.
  */
-function gutenberg_remove_wpcom_markdown_support( $post ) {
-	if ( gutenberg_content_has_blocks( $post->post_content ) ) {
-		remove_post_type_support( 'post', 'wpcom-markdown' );
+function gutenberg_remove_wpcom_markdown_support( $post_data ) {
+	if ( gutenberg_content_has_blocks( implode( "\n", $post_data ) ) ) {
```

Comment:
What is the shape of `$post_data`? From what I can tell, it's a compacted associative array of all properties:

https://github.com/WordPress/wordpress-develop/blob/fde455a89f9eaaae9513ca722f2de20420ca4bdd/src/wp-includes/post.php#L3487

Why do we test anything other than content?

Reply:
Fixed. I misread documentation that post_data was just the post_content in an array.  I'm not sure what I did on my initial inspection of the data to convince me that was the case.